HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -33,6 +33,8 @@ public record Questionnaire
     public string Hug2ReferralId { get; set; }
     
     public string LaContactName { get; set; }
+    public bool? LaCanContactByEmail { get; set; }
+    public bool? LaCanContactByPhone { get; set; }

```

Comment:
Would this be simpler as a derived property? (Is it ever not `!string.IsNullOrEmpty(LaContactTelephone)`)

Reply:
It will be null the first time the user views the page - that way we won't pre-fill the radio buttons